src/NexusMods.CrossPlatform/NexusMods.CrossPlatform.csproj

```diff
@@ -42,4 +42,8 @@
       </Compile>
     </ItemGroup>
 
+    <ItemGroup>
+      <PackageVersion Update="CliWrap" Version="3.6.6" />
+    </ItemGroup>
```

Comment:
This seems very random. Probably should be removed.

Reply:
Yep was added through debugging some of the compile time failures, removed.